app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -122,13 +122,16 @@ def permission_template
       Hyrax::PermissionTemplate.find_by!(source_id: id)
     end
 
-    # Calculate and update who should have edit access based on who
-    # has "manage" access in the PermissionTemplateAccess
+    # Calculate and update who should have read/edit access to the collections based on who
```

Comment:
It looks to me that this method overwrites any preexisting access controls assigned to this collection.  Is this the intended behavior?  If so, can we document that here?

Reply:
Discussed in slack and the concern is about the name of the method not clearly conveying what the method is doing.  Changing name to reset_access_controls!

Reply:
Request to change the name was created in issue #2455.